scrapyloganalyzer/__init__.py

```diff
@@ -199,4 +199,7 @@ def error_rate(self) -> float:
            containing 99 release packages.
         """
         error_count = self.item_counts["FileError"] + self.logparser["crawler_stats"].get("invalid_json_count", 0)
-        return error_count / (self.item_counts["File"] + self.item_counts["FileItem"] + error_count)
+        items_count = self.item_counts["File"] + self.item_counts["FileItem"]
+        if items_count:
+            return error_count / (items_count + error_count)
```

Comment:
I think more robust to `import contextlib` and then:

```suggestion
        with contextlib.suppress(ZeroDivisionError)
            return error_count / (self.item_counts["File"] + self.item_counts["FileItem"] + error_count)
```